webpack.config.js

```diff
@@ -1,127 +1,81 @@
 
 module.exports = {
-    entry: [`${__dirname}/js/main.js`, `${__dirname}/scss/app.scss`],
-
     output: {
         path: `${__dirname}/dist`,
-        filename: "js/main.js",
-    },
-    
-
-    devServer: {
-        hot: true,
-        static: "./dist",
+        clean: true,
     },
 
-    devtool: devMode ? "inline-source-map" : false,
+    devtool: isDevel ? 'inline-source-map' : false,
 
     performance: {
         hints: false,
     },
 
+    resolve: {
+        alias: {
+            // (best practice) webpack aliases can be used in HTML, SCSS, JS to avoid relative paths like `../../images/`
+            '@images': path.join(__dirname, 'src/images'),
+            '@fonts': path.join(__dirname, 'src/fonts'),
+            '@scripts': path.join(__dirname, 'src/js'),
+            '@styles': path.join(__dirname, 'src/scss'),
+        },
+    },
+
     module: {
         rules: [
             {
                 test: /\.js?$/,
                 exclude: /(node_modules|bower_components)/,
-                loader: "babel-loader",
+                loader: 'babel-loader',
                 options: {
-                    presets: ["@babel/preset-env"],
+                    presets: ['@babel/preset-env'],
                 },
             },
             {
                 test: /\.scss$/,
-                use: [
-                    {
-                        loader: MiniCssExtractPlugin.loader,
-                        options: {
-                            publicPath: "../",
-                        },
-                    },
-
-                    {
-                        loader: "css-loader",
-                        options: { sourceMap: devMode, url: true },
-                    },
-                    {
-                        loader: 'resolve-url-loader',
-                        options: { sourceMap: true },
-                    },
-                    {
-                        loader: "sass-loader",
-                        options: { sourceMap: true },
-                    },
-                ],
-            },
-            {
-                test: /\.pug$/,
-                use: {
-                    loader: "@webdiscus/pug-loader",
-                },
+                use: [ 'css-loader',  'sass-loader'],
             },
             {
                 test: /\.(jpg|png|gif|svg|woff|woff2|eot|ttf|otf)$/i,
                 type: 'asset/resource',
                 generator: {
                     filename: 'images/[name][ext]'
                 },
-                // use: [
-                //     {
-                //         loader: "file-loader",
-                //         options: {
-                //             name: "images/[name].[ext]",
-                //         },
-                //     }
-                // ],
             },
         ],
     },
 
     plugins: [
-        new CleanWebpackPlugin(),
-        new ErrorNotificationPlugin(),
-        browserSync,
```

Comment:
instead of outdated  `browser-sync-webpack-plugin` use the `devServer.watchFiles` option to live reload browser after changes